### ISSUE TITLE
No more values are displayed

### ISSUE THREAD

**Issue by ImhotepDE** [tolwi/hassio-ecoflow-cloud#340]
Hello,

No more values are displayed. Installation did not help.

Thanks

**Comment by YSmetana** [comment]
Have you logged into the Ecoflow app on your smartphone recently?

**Comment by ImhotepDE** [comment]
Yes, I do this every day, it went without problems the last few months
